### ISSUE TITLE
Create slot for summary fig caption in charts

### ISSUE THREAD

**Issue by Lucas1981** [Adyen/lume#164]
Right now there is no way for people who cannot read the screen to know what a chart is showing. Providing a figcaption slot would solve this problem, where consumers could provide a summary description of what the chart is showing for their users.
